### ISSUE TITLE
__typename is not returned when queried

### ISSUE THREAD

**Issue by cwheel** [weiznich/wundergraph#57]
In a query like the following:

```gql
query {
   Hero {
      __typename
   }
}
```

The expected response would contain a key like:

```gql
"__typename": "Hero"
```

Instead, the `__typename` key is omitted from the response. I've done a bit of poking around in Juniper, and it seems that the `concrete_type_name` function is called to determine the value `__typename` should resolve when queried. Wundergraph appears to define that function [here](https://github.com/weiznich/wundergraph/blob/2e2ccf377d23dafaa7106e47c454f5b5aedd1307/wundergraph/src/macros/query.rs#L639), but to no effect. Any thoughts on what might be happening here? Thanks!

**Comment by p-alik** [comment]
`Hero` expects a `primaryKey` parameter:

![hero](https://user-images.githubusercontent.com/2326795/89528797-f40cfe00-d7eb-11ea-99fe-f475e8a4510c.png)

~~`concrete_type_name` is unused yet.~~

**Comment by cwheel** [comment]
Right, that query above was a (poor) example. However, in the example you provided, I'd expect the response to be:

```gql
"data": {
   "Hero": {
       "__typename": "Hero"
    }
}
```
I'm also not quite sure I understand what you mean by being unused. As I understand it, `concrete_type_name` is something Juniper expects to be implemented.

**Comment by p-alik** [comment]
> I'm also not quite sure I understand what you mean by being unused. As I understand it, `concrete_type_name` is something Juniper expects to be implemented.

Please, ignore my silly remark regarding `concrete_type_name`

**Comment by cwheel** [comment]
Got it, so if Wundergraph is implementing `concrete_type_name`, shouldn't  Juniper be able to resolve `__typename`?
